### ISSUE TITLE
cffconvert fails conversion of invalid CITATION.cff but returns 0 

### ISSUE THREAD

**Issue by abelsiqueira** [citation-file-format/cff-converter-python#266]
**Reproduction steps:**

I have a wrong `CITATION.cff` (typo on key). I run `cffconvert -f zenodo` and get
```
'CITATION.cff' does not pass validation. Conversion aborted.
```
But the return value is `0`. Verified with `echo $?`.

**Comment by jspaaks** [comment]
Probably somehting to do with this code here

https://github.com/citation-file-format/cff-converter-python/blob/c0953fcc7ad4a9b5711b8562465eb2bb0ec94487/cffconvert/cli/validate_or_write_output.py#L21-L26

Might need to be `ctx.exit(code=1)` or something.

**Comment by jspaaks** [comment]
Fixed in PR #320, closing
